package.json

```diff
@@ -10,7 +10,7 @@
     "test:coverage": "jest --coverage"
   },
   "dependencies": {
-    "@reportportal/client-javascript": "~5.3.0",
+    "@reportportal/client-javascript": "^5.4.0",
```

Comment:
Please do not bump client version to minor without intendance to release the next version as minor. (I hope this bugfix should be released as patch version first).
The `~` used here to lock auto-updates to minor client versions as they may contain breaking changes according to our release approach.
I'll add more details to the KB on this decision.